src/vs/workbench/services/editor/common/editorService.ts

```diff
@@ -80,7 +80,15 @@ export interface IBaseSaveRevertAllEditorOptions {
 	/**
 	 * Whether to include untitled editors as well.
 	 */
-	readonly includeUntitled?: boolean;
+	readonly includeUntitled?: {
```

Comment:
Could we try to have a unit test for this? I guess save will be tricky because it will open a dialog but maybe revert could be tested? With your recent changes it should close a scrapbook interactive editor I suppose.

Reply:
the `TestFileEditorInput` already just had a spy method stubbed for save, so no need to worry about the dialog